.github/workflows/leo-build-tag-publish-and-run-tests.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           inputs: '{
               "repository": "${{ github.event.repository.full_name }}",
               "ref": "${{ needs.init-github-context.outputs.build-branch }}",
-              "leonardo-release-tag": "${{ steps.tag.outputs.tag }}"
+              "leonardo-release-tag": "${{ steps.tag.outputs.tag }}-${{ needs.init-github-context.outputs.build-branch }}"
```

Comment:
me, personally, I would _not_ add the branch name to the `leonardo-release-tag`. it looks like that version is going to find its way into [helm charts and a bunch of other downstream systems](https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/leonardo-build.yaml#L114-L146) (not sure if that is intended).

the branch name is also in the "ref" input above.
I would modify the [leonardo-build.yaml](https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/leonardo-build.yaml#L97-L101) with the a tag for `${{ inputs.ref }}`

Reply:
Yep, this is the consensus from Devops as well. I will work on it when I have some focus time this week :)